database/elastic_mapping.js

```diff
@@ -5,6 +5,7 @@ import languages from '../app/shared/languages';
 
 const config = {
   settings: {
+    'index.mapping.total_fields.limit': 2000,
```

Comment:
Do we really want a global upping of this limit, or on a 'per collection' case whenever needed?

As I understand it, this could impact performance or, at the very least, resource allocation, right?

Reply:
The documentation says the limit is there to prevent mappings from becoming too large and impact performance, but it's not clear if just by increasing the limit it impacts the performance, or increasing the limit and mappings becoming too large then is what impacts the performance.
So maybe the instances that do not use the extra limit won't add performance problems.
It would be great to have a way of measuring this.